web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -836,6 +836,16 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
    */
   @volatile var cometRenderTimeout: Long = 30.seconds
 
+  /**
+    * Adjusts LiftRules to allow a page's comets to "rehydrate" (i.e. reconnect to the page) after a server restart.
+    * This works by reloading the page into an iframe and stealing all of the comets. When the refresh occurs,
+    * new comet actors will be instantiated, so beware that any state therein will NOT be reinstated.
+    */
+  def enableCometRehydration(): Unit = {
+    LiftRules.redirectAsyncOnSessionLoss = false
+    LiftRules.noCometSessionCmd.default.set(() => JsCmds.Run("lift.rehydrateComets()"))
+  }
+
```

Comment:
@farmdawgnation @Shadowfiend I feel like the hardest part to get right here is the comment...

Reply:
I think that the comment strikes the right chord overall. The one guidance we might add would be to say that you should code applications depending on this behavior to be able to reconstruct comet state from cookies or from information available from the client (e.g. JS state, state stores, etc)

If you feel like that would be a good addition I say add it. Otherwise, I'm going to approve this PR.

Reply:
Comment updated